input/resources/StructureDefinition-indv-measurereport-deqm.xml

```diff
@@ -207,11 +207,20 @@
     </element>
     <element id="MeasureReport.measure">
       <path value="MeasureReport.measure"/>
+      <short value="What measure and version was calculated"/>
+      <definition value="A reference to the Measure that was calculated to produce this report."/>
       <requirements value="This will indicate the measure being reported on. For example, the cannonical uri location for the MRP measure."/>
       <type>
         <code value="canonical"/>
         <targetProfile value="http://hl7.org/fhir/uv/crmi/StructureDefinition/crmi-shareablemeasure"/>
       </type>
+      <condition value="deqm-0"/>
```

Comment:
Tested by removing version "|5.0.0" from the MeasureReport-indv-measurereport01.json file, see below
"measure": "http://hl7.org/fhir/us/davinci-deqm/Measure/measure-mrp-example",
It does not look like the deqm-0 constraint is triggered. QA report shows 0 errors/warnings for this file.